release-notes/release-notes5106.md

```diff
@@ -0,0 +1,142 @@
+---
+
+## Overview
+
+{{site.productname}} 5.10.6 was released for {{site.enterpriseversion}} and {{site.cloudname}} on October 26<sup>th</sup>, 2022. It includes {{site.productname}} 5.10.6. These release notes provide an overview of the changes for {{site.productname}} 5.10.6, including:
+
+- [Accompanying Premium Plugin changes](#accompanyingpremiumpluginchanges)
+- [Accompanying Premium self-hosted server-side component changes](#accompanyingpremiumself-hostedserver-sidecomponentchanges)
+- [General Bug fixes](#bugfixes)
+- [Upgrading to the latest version of TinyMCE 5](#upgradingtothelatestversionoftinymce5)
+
+{{site.releasenotes_for_stable}}
+
+## Accompanying Premium Plugin changes
+
+The following premium plugin updates were released alongside {{site.productname}} 5.10.6.
+
+### PowerPaste 5.6.2
+
+The {{site.productname}} 5.10.6 release includes an accompanying release of the **PowerPaste** premium plugin.
+
+#### If URL paths contained either a colon or an exclamation point (or both) PowerPaste’s link detection logic failed
+
+If a colon or an exclamation point — `:` or `!` — was present in the path portion of a URL, PowerPaste’s link detection logic treated the characters as, in effect, end-of-path boundaries. As a consequence, while the string was pasted into {{site.productname}}, the expected link was incomplete and incorrect.
+
+For example, pasting the string
+
+`https://example.com/!exclamation-point`
+
+resulted in the following, incomplete, link and trailing text
+
+```html
+<a href="https://example.com/">https://example.com/</a>!exclamation-point
+```
+Equivalent results occurred when URL paths contained the colon character (for example, `https://example.com/:colon`).
+
+With this update, the link detection logic has been improved to capture and present such URLs correctly.
+
+Now, pasting the string 
+
+`https://example.com/!exclamation-point`
+
+results in the following, complete, link and no trailing text
+
+```html
+<a href="https://example.com/!exclamation-point">https://example.com/!exclamation-point</a>
+```
+
+> **NOTE**: Exclamation points and colons are still boundary characters for URL detection and delimiting. If the string `https://example.com/colon:` is pasted or typed into a {{site.productname}} document, automatic link detection will produce a link that does not include the trailing colon character.
+
+For information on the PowerPaste plugin, [PowerPaste plugin]({{site.baseurl}}plugins/premium/powerpaste/).
+
+
+## Accompanying Premium self-hosted server-side component changes
+
+The {{site.productname}} 5.10.6 release includes changes affecting the {{site.productname}} **self-hosted** services for the following plugins:
+
+- The Enhanced Media Embed plugin (`mediaembed`)
+- The Image Tools plugin (`imagetools`)
+- The Link Checker plugin (`linkchecker`)
+- The Spell Checker Pro plugin (`tinymcespellchecker`)
+
+The Java server-side components have been updated to the following versions:
```

Comment:
@brian-forte Here are the version numbers for services based on BOM (https://ephocks.atlassian.net/browse/TINY-8884) & 5.10.6 RC package verification by QA -

**Services**

- Hyperlinking 2.105.13
- Image proxy 2.106.4
- Spelling 2.118.6
 
Can you please check and make changes accordingly?

Reply:
> Can you please check and make changes accordingly?

Checked. Corrected one version number string. The text is now as follows:

```asciidoc
- `ephox-spelling.war`: 2.118.6
- `ephox-hyperlinking.war`: 2.105.13
- `ephox-image-proxy.war`: 2.106.4
```